src/components/button/styles.css

```diff
@@ -37,31 +37,33 @@
 	color: #1873b4;
 	border: 2px solid #1873b4;
 	background-color: transparent;
-	}
-	.button.color_transparent:hover {
-		color: #fff;
-		background-color: #1873b4;
-		}
+}
+.button.color_transparent:hover {
+	color: #fff;
+	background-color: #1873b4;
+}
 .button.mod_download {
-	}
-.button.mod_download:not(:first-child) {
-	margin-left: 20px;
-	}
-	.button.mod_download:hover .button__icon g {
-		fill: #fff;
-		}
-	.button.mod_download:hover .button__version {
-		color: rgba(255, 255, 255, 0.5);
-		}
+	margin-bottom: 7px;
```

Comment:
@Michadelic, I'm sorry, but now buttons are not aligned by vertical center in the block. This bottom margin should be compensated by inner block padding of a blue block:
![incorrect_margin](https://user-images.githubusercontent.com/748043/53027703-c00e1380-3465-11e9-9545-fa9dea207042.png)